ocaml/xapi/xapi_xenops.ml

```diff
@@ -695,9 +695,8 @@ module MD = struct
   let vusbs_of_vm ~__context (vmref, vm) =
     vm.API.vM_VUSBs
     |> List.map (fun self -> Db.VUSB.get_record ~__context ~self)
-    (* get attached PUSBs. *)
-    |> List.filter (fun vusb -> vusb.API.vUSB_attached <> Ref.null)
-    |> List.map (fun vusb -> Db.PUSB.get_record ~__context ~self:vusb.API.vUSB_attached)
+    |> List.map (fun vusb -> Db.USB_group.get_record ~__context ~self:vusb.API.vUSB_USB_group)
+    |> List.map (fun usb_group -> Db.PUSB.get_record ~__context ~self:(List.hd usb_group.API.uSB_group_PUSBs))
```

Comment:
This one needs to be caught, and an Internal_error raised saying that there is no PUSB associated with the USB_group

Reply:
Ok

Reply:
updated.